src/helix-ui/styles/elements/hx-tab.less

```diff
@@ -18,11 +19,16 @@ hx-tab {
 
   &:active {
     color: @cyan-700;
+    z-index: 1;
   }
   
   &[current] {
-    background-color: inherit;
+    background-color: @gray-25;
```

Comment:
@palindromebart @racksmangonzalez: Design Review Requested

We changed the `tab` default `background-color` to **gray-25** so that the `tab` is highlighted when selected.  Any thoughts?

<img width="1421" alt="pr324-1" src="https://user-images.githubusercontent.com/10120600/45115453-7a5aca00-b115-11e8-9785-cda85823816d.png">

Reply:
Reverting changes